apps/charterafrica/src/lib/data/common/index.js

```diff
@@ -124,7 +120,10 @@ export async function getPageProps(api, context) {
     return null;
   }
 
-  const [page] = pages;
+  let [page] = pages;
+  if (params?.slugs?.length === 3) {
+    page = await pagify(page, api, context);
```

Comment:
There's a possibility of pagify returning `null`.

Setting page.blocks on null will throw an error

Reply:
Nice catch. `null` would mean `notFound`. Let me push a fix.